app/test/unit/parse-files-to-be-overwritten-test.ts

```diff
@@ -0,0 +1,101 @@
+import { setupEmptyRepository } from '../helpers/repositories'
+import { writeFile } from 'fs-extra'
+import { join } from 'path'
+import { git } from '../../src/lib/git'
+import { cloneLocalRepository } from '../helpers/repository-scaffolding'
+import { GitError as DugiteError } from 'dugite'
+import { parseFilesToBeOverwritten } from '../../src/ui/lib/parse-files-to-be-overwritten'
+
+describe('parseFilesToBeOverwritten', () => {
+  it('parses files from pull error', async () => {
```

Comment:
These tests are awesome! Thanks for taking care of calling the actual git commands here ❤️

Reply:
Thanks, it'd be nice if this parsing (and testing) would live in dugite instead but we'll save that for another day